src/js/video.js

```diff
@@ -129,7 +129,7 @@ setup.autoSetupTimeout(1, videojs);
  *
  * @type {String}
  */
-videojs.VERSION = '__VERSION__';
+videojs.VERSION = require('../../package.json').version;
```

Comment:
I assume this is where the babel plugin comes in, my question is: Why even bother trying to make it a literal? We could just use it like this and have rollup shake off everything from the package.json that we don't use (effectively making it literal anyway).